src/rules/mongodb.js

```diff
@@ -9,6 +9,8 @@
  */
 
 function (user, context, callback) {
+  const mongo = require('mongo');
```

Comment:
this shouldn't be necessary with rules, since `mongo` is injected as a magic global
https://github.com/auth0/auth0-authz-rules-api/blob/master/index.js#L62-L70